packages/apollo-server-errors/src/index.ts

```diff
@@ -129,10 +164,13 @@ export function fromGraphQLError(error: GraphQLError, options?: ErrorOptions) {
 
   // copy enumerable keys
   Object.entries(error).forEach(([key, value]) => {
+    if (key === 'extensions') {
+      return; // extensions are handeled bellow
+    }
     copy[key] = value;
   });
 
-  // extensions are non enumerable, so copy them directly
+  // merge extensions instead of just copying them
```

Comment:
I guess this affects any extensions that are assigned in the constructor?

Reply:
yes, v16 just makes `extensions` always be enumerable even if empty.